test/e2e/framework/log_test.go

```diff
@@ -41,6 +35,12 @@ import (
 //
 //
 //
+//
+//
+//
+//
+//
+// This must be line #43 (first line is #1).
```

Comment:
What does `first line is #1` mean?

Reply:
That my editor is counting from 1. I'm not sure whether all of them do (first line might also be #0). Hmm, compilers and linters all print line numbers. I probably was too cautious here, there is an established convention. Let me remove it.